### ISSUE TITLE
Fix out of box installation

### ISSUE THREAD

**Issue by vivek378521** [tursodatabase/turso#71]
If the rust version on your machine is <= 1.73 then the `make` commands throws an error

`error: package `clap_derive v4.5.5` cannot be built because it requires rustc 1.74 or newer, while the currently active rustc version is 1.73.0`

After you update rust to the newest version and run `make`, it throws another error

```
error[E0463]: can't find crate for `core`
  |
  = note: the `wasm32-wasi` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-wasi`

For more information about this error, try `rustc --explain E0463`.
error: could not compile `siphasher` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...

```

You can fix this error by running `rustup target add wasm32-wasi`

In an ideal scenario, all external dependencies should be downloaded when one runs the `make` command.

**Comment by vivek378521** [comment]
Raised a PR for this: https://github.com/penberg/limbo/pull/99

I also think that people might find it annoying if we just update their rust versions, so let me know please if this works or suggest what should I add more. 

Thanks!

**Comment by penberg** [comment]
Fixed by 014f3c457a9520eec0bcd05b894c08bd39be7cd3
